tools/gromacs/sim.xml

```diff
@@ -98,16 +98,28 @@
                     pme_order      = 4        ; cubic interpolation
                     fourierspacing  = 0.16    ; grid spacing for FFT
                     ; Temperature coupling is on
+                    #if $sets.mdp.couplinggroups.wholesystem =="true":
+                    tcouple   = V-recale ; modified Berendsen thermostat
```

Comment:
```suggestion
                    tcouple   = V-rescale ; modified Berendsen thermostat
```
move outside the `if`?

Reply:
sure this could work. this should be constant either way.